xla/backends/gpu/runtime/collective_thunk.cc

```diff
@@ -58,6 +58,10 @@ limitations under the License.
 #include "xla/tsl/platform/statusor.h"
 #include "xla/util.h"
 
+// TODO: These timeouts should really be configurable via debug options.
```

Comment:
Can you add proper options for that, extern "C" "options" are not used anywhere else in XLA and it will be a very bad example.

Reply:
I think @giordano might need some help / pointers here (they're not familiar with the XLA codebase at all), if you can point to what would need to be updated

Reply:
It should be added to this section in the proto: https://github.com/openxla/xla/blob/main/xla/xla.proto#L231

And flag parsing initialized in: https://github.com/openxla/xla/blob/main/xla/debug_options_flags.cc